lib/workers/branch/__snapshots__/commit.spec.ts.snap

```diff
@@ -13,6 +13,7 @@ Array [
       ],
       "force": false,
       "message": "some commit message",
+      "noVerify": undefined,
```

Comment:
Shouldn't this have a default value?

Reply:
For this test case, no param was passed to `noVerify`, which means it'll get the `undefined` value. I don't think the snapshotting can take into account default func args (e.g. what `undefined` gets turned into later)?

Reply:
yes, so it can happen that a passed undefined is overriding the function default value, so make sure the parameter is not undefined, eg: `noVerify?.includes('commit')`

Reply:
I think TS accepts it because it sees that the value is never actually undefined when used.
E.g. this statement:
```ts
const { noVerify = [] } = { noVerify: undefined };
```
yields the same result as
```ts
const { noVerify = [] } = {};
```

Reply:
So removing the default value doesn't really gain us anything, other than the ability to do explicit checks on whether a value was not passed (but we can't do param destructuring if we wanna do that)?

Reply:
But if you prefer the no-default-but-still-optional route that's fine, just lmk.

Reply:
Destruction is possible, so let's go with optional and non default parameter as my latest suggestions. 🤗